styles/components/card.css

```diff
@@ -0,0 +1,57 @@
+.card {
+  border: 1px solid black;
+  border-radius: 10px;
+  position: relative;
+  background-color: lightgoldenrodyellow;
+  height: max-content;
+  max-width: 400px;
+  margin: 30px auto;
+  xdisplay: flex;
+  xflex-direction: column;
+}
+
+.card__bookmark {
+  position: absolute;
+  right: 20px;
+  top: -10px;
+  border-radius: 5px;
+}
+
+.card__heading {
+  margin-left: 10px;
+  font-family: Georgia, "Times New Roman", Times, serif;
+}
+
+.card__question {
+  margin: 5px 10px 20px;
+}
+
+.card__summary {
+  margin: 10px auto 15px;
+  padding: 5px;
+  text-align: center;
+  list-style-type: none;
+  border: 1px solid black;
+  width: max-content;
+  background-color: lightgrey;
+  border-radius: 10px;
+}
+
+.card__summary:hover {
+  background-color: lightsalmon;
+  color: white;
+}
+
+.card__ul {
```

Comment:
For the accessibility you can you can use role="list" in your HTML-file and .card__ul[role="list"] in your CSS-file